chargeOclock/devTemp/devClientsViewController.swift

```diff
@@ -47,4 +51,32 @@ class devClientsViewController: UITableViewController {
 		}
 	}
 
+	func plusTapped() {
+
+		let alert = UIAlertController(title: "Add Client 👔", message: "Please, enter client name 😉", preferredStyle: .alert)
+
+		alert.addTextField { textField in
+			textField.placeholder = "Client name"
+		}
+
+		alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil ))
+
+		alert.addAction(UIAlertAction(title: "Create", style: .default) { [weak alert] (_) in
+
+			guard let name = alert?.textFields?.first?.text else {
+				return
+			}
+
+			Client.addClient(name: name) { error, data in
+				if error != nil {
+					print("Error")
+				} else if data != nil {
+					self.refresh(refreshControl: self.tableView.refreshControl ?? UIRefreshControl() )
```

Comment:
hm... 🤔 don't really like the `?? UIRefreshControll()`, but don't have a good alternative for now, so leave